Documentation/release-notes/aapt.md

```diff
@@ -0,0 +1,19 @@
+#### Deprecation of aapt in favor of aapt2
+
+* warning XA1026: Using 'aapt' is deprecated in favor of 'aapt2'.
+  Please enable 'Use incremental Android packaging system (aapt2)' in
+  the Visual Studio project property pages or edit the project file in
+  a text editor and set the 'AndroidUseAapt2' MSBuild property to
+  'true'.
+
+[Google has deprecated][aapt] the `aapt` command-line tool in favor of
+`aapt2` going forward. Xamarin.Android has now deprecated
+`<AndroidUseAapt2>false</AndroidUseAapt2>` as well.
+
+Update the `$(AndroidUseAapt2)` MSBuild property to `true` to select
+'aapt2'. This property corresponds to the **Use incremental Android
+packaging system (aapt2)** setting in the Visual Studio project
+properties pages. Alternatively, remove `<AndroidUseAapt2>` from the
+_.csproj_ file to let the build select the default value `true`.
+
+[aapt]: https://developer.android.com/studio/command-line/aapt2#aapt2_changes
```

Comment:
We should probably warn users that changing to aapt2 might well break their apps. Since aapt2 is stricter than aapt.
We could also advise users to use ` --legacy` in the `AndroidAapt2CompileExtraArgs` so that those newer errors can be treated as warnings until they can fix them.

Reply:
In case it might be handy as a starting place or a verbatim copy-paste, the Xamarin.Android 10.0 release notes used the following text:

```markdown
> [!IMPORTANT]
> AAPT2 will in some cases enforce stricter rules on resource files than the
> previous AAPT, so some adjustments might be needed if you see new error
> messages that come from AAPT2 itself rather than from the Xamarin.Android
> build tasks.
```